lxd/api_internal.go

```diff
@@ -133,6 +135,18 @@ var internalWarningCreateCmd = APIEndpoint{
 	Post: APIEndpointAction{Handler: internalCreateWarning},
 }
 
+var internalRecoverValidateCmd = APIEndpoint{
+	Path: "recover/validate",
+
+	Post: APIEndpointAction{Handler: internalRecoverValidate},
+}
+
+var internalRecoverImportCmd = APIEndpoint{
+	Path: "recover/import",
+
+	Post: APIEndpointAction{Handler: internalRecoverImport},
+}
+
```

Comment:
Given how many functions and code is in this file, it may make sense to move to a api_internal_recover.go file